backend/src/database/sql/migrations/20241115064625-create-author.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+import { QueryInterface, DataTypes } from 'sequelize';
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface: QueryInterface, Sequelize: typeof DataTypes) {
+    await queryInterface.createTable('Authors', {
```

Comment:
[Nit] It is customary to name models as the singular entity. So "Author", "Book" etc.